### PR TITLE
[EDU-3924] feat: metrics requests edge app

### PR DESCRIPTION
**Pull request by hannah-sk**
### Changes

* Added Billing section to Edge Application reference.
* Added info on Requests metric.

[EDU-3924]: https://aziontech.atlassian.net/browse/EDU-3924?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/content/docs/en/pages/main-menu/reference/build/edge-application/edge-application.mdx [hunk 1]
```diff
@@ -119,6 +119,23 @@ These are the **default limits** for each Service Plan:
...
+
+## Billing
+
+**Edge Application** has specific metrics that define how the product is billed.
+
+<Button href="https://www.azion.com/en/pricing/#EdgeApplication" text="see pricing"></Button>
+
+### Requests
+
+Total amount of HTTP and HTTPS requests from an user or device made to edge applications running on Azion Edge Computing Platform. A request is billed independently of request success or failure and version of server protocol (HTTP/1.1, HTTP/2.0, HTTP/3.0).
```
Comment (bru-andrade) [inline src/content/docs/en/pages/main-menu/reference/build/edge-application/edge-application.mdx:132]:
```suggestion
Total amount of HTTP and HTTPS requests from an user or device made to edge applications running on Azion Edge Computing Platform. A request is billed independently of request success or failure and server protocol version (HTTP/1.1, HTTP/2.0, HTTP/3.0).
```

File: src/content/docs/en/pages/main-menu/reference/build/edge-application/edge-application.mdx [hunk 2]
```diff
@@ -119,6 +119,23 @@ These are the **default limits** for each Service Plan:
...
+
+When a content, such as an image cached at the edge is requested, it isn't billed.
+
+When a request using [Image Processor](/en/documentation/products/build/edge-application/image-processor/) is processed, it counts as one request independently of the amount of filters being used. For example, if the request queries three filters, it'll be billed as one request.
```
Comment (bru-andrade) [inline src/content/docs/en/pages/main-menu/reference/build/edge-application/edge-application.mdx:136]:
```suggestion
When a request using [Image Processor](/en/documentation/products/build/edge-application/image-processor/) is processed, it counts as one request independently of the amount of query strings being used to apply filters. For example, if the request queries three filters, it'll be billed as one request.
```

Each filter is a query string (`?filter=value&filter2=value2`), maybe this is clearer.

File: src/content/docs/en/pages/main-menu/reference/build/edge-application/edge-application.mdx [hunk 3]
```diff
@@ -119,6 +119,23 @@ These are the **default limits** for each Service Plan:
...
+
+**Edge Application** has specific metrics that define how the product is billed.
+
+<Button href="https://www.azion.com/en/pricing/#EdgeApplication" text="see pricing"></Button>
+
+### Requests
+
+Total amount of HTTP and HTTPS requests from an user or device made to edge applications running on Azion Edge Computing Platform. A request is billed independently of request success or failure and version of server protocol (HTTP/1.1, HTTP/2.0, HTTP/3.0).
+
+When a content, such as an image cached at the edge is requested, it isn't billed.
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/main-menu/reference/build/edge-application/edge-application.mdx:134]:
```suggestion
When content, such as an image cached at the edge, is requested, it isn't billed.
```

File: src/content/docs/en/pages/main-menu/reference/build/edge-application/edge-application.mdx (hunk 2 above)
Comment (MarianaAguilera) [inline src/content/docs/en/pages/main-menu/reference/build/edge-application/edge-application.mdx:136]:
```suggestion
When a request using [Image Processor](/en/documentation/products/build/edge-application/image-processor/) is processed, it counts as one request independently of the number of filters being used. For example, if the request queries three filters, it's billed as one request.
```

**Review by MarianaAguilera — APPROVED** [review]

**Pull request merged**